runtime/autoload/help.vim

```diff
@@ -0,0 +1,38 @@
+function! s:synnames(...) abort
+  if a:0
+    let [line, col] = [a:1, a:2]
+  else
+    let [line, col] = [line('.'), col('.')]
+  endif
+  return reverse(map(synstack(line, col), 'synIDattr(v:val,"name")'))
+endfunction
+
+function! HelpTopic()
```

Comment:
Since you are using autoload file you must use `help#topic()` (`help` is because file is named `{rtp}/autoload/help.vim`).